docs/ruleset_functions/G311_exceptions/zone_meets_G3_1_1c

```diff
@@ -0,0 +1,74 @@
+# zone_meets_G3_1_1c
+
+**Schema Version:** 0.0.21
+
+**Description:** determines whether a given zone meets the G3_1_1c exception "If the baseline HVAC system type is 5, 6, 7, 8, 9, 10, 11, 12, or 13 use separate single-zone systems conforming with the requirements of system 3 or system 4 (depending on building heating source) for any spaces that have occupancy or process loads or schedules that differ significantly from the rest of the building. Peak thermal loads that differ by 10 Btu/h·ft2 (2.930710 W/sf) or more from the average of other spaces served by the system, or schedules that differ by more than 40 equivalent full-load hours per week from other spaces served by the system, are considered to differ significantly. Examples where this exception may be applicable include but are not limited to natatoriums and continually occupied security areas. This exception does not apply to computer rooms."
+"average" is not particularly well defined in this exception. 
+for the peak W/sf calculation I've assumed:
+	1. the zone being tested is not included
+ 	2. the average: (each space coincident peak W) / total_area
+
+**Inputs:** 
+- **RMR**
```

Comment:
Should we start calling it RMD? (I think Christina also keeps using RMR abbreviation.)
Also, we need to indicate that this function applies to the B_RMD, not P_ or U_

Reply:
ok, will change